.pre-commit-config.yaml

```diff
@@ -29,3 +29,11 @@ repos:
     hooks:
     -   id: mypy
         args: [--no-strict-optional, --ignore-missing-imports]
+-   repo: local
+    hooks:
+    -   id: check-rebase
+        name: Check if rebase is needed
+        entry: ./files/scripts/check-rebase.py
+        language: script
+        types: [python]
```

Comment:
I think we can remove this since we have `always_run: true`. (And this may be a reason of the duplication of the output.)